frc46_token/src/token/state.rs

```diff
@@ -627,14 +687,22 @@ pub fn decode_actor_id(key: &BytesKey) -> Option<ActorID> {
 
 impl Cbor for TokenState {}
 
+/// A summary of the current state to allow checking application specific invariants
+#[derive(Clone, Debug)]
+pub struct StateSummary {
+    pub balance_map: HashMap<ActorID, TokenAmount>,
+    pub allowance_map: HashMap<ActorID, BTreeMap<ActorID, TokenAmount>>,
```

Comment:
```suggestion
    pub allowance_map: HashMap<ActorID, HashMap<ActorID, TokenAmount>>,
```
You copied this from mistaken code that I wrote